### PR TITLE
UI/license autoload

### PR DESCRIPTION
**Pull request by hashishaw**
Update license page for autoloaded license.

**When Autoloaded**
Screen Shot 2021-06-03 at 4 37 24 PM
**Still shows standby count**
Screen Shot 2021-06-03 at 4 36 41 PM
**Shows warning if license is stored**
Screen Shot 2021-06-03 at 4 36 39 PM

### CODE REVIEW

File: ui/app/components/license-info.js [hunk 1]
```diff
@@ -1,22 +1,32 @@
-import { equal } from '@ember/object/computed';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
```
Comment (Monkeychip) [inline ui/app/components/license-info.js:1]: nice work glimmerizing

File: ui/app/templates/components/license-info.hbs [hunk 2]
```diff
@@ -3,72 +3,30 @@
...
-      </form>
-    {{else}}
-      <div class="field box is-fullwidth is-shadowless is-paddingless is-marginless">
-        <InfoTableRow @label="License ID" @value={{licenseId}} />
-        <InfoTableRow @label="Valid from" @value={{startTime}}>
-          {{date-format startTime 'MMM dd, yyyy hh:mm:ss a'}} to {{date-format expirationTime 'MMM dd, yyyy hh:mm:ss a'}}
+
+<section class="box is-sideless is-marginless is-shadowless is-fullwidth">
+  <span class="title is-5">Details</span>
+
```
Comment (Monkeychip) [inline ui/app/templates/components/license-info.hbs:9]: is it intentional leaving a line space here?

File: ui/mirage/config.js [hunk 3]
```diff
@@ -29,6 +31,33 @@ export default function() {
...
+    return {
+      autoloading_used: false,
+      stored: {
+        expiration_time: EXPIRY_DATE,
+        features: ['DR Replication', 'Namespaces', 'Lease Count Quotas', 'Automated Snapshots'],
+        license_id: '0eca7ef8-ebc0-f875-315e-3cc94a7870cf',
+        performance_standby_count: 0,
+        start_time: '2020-04-28T00:00:00Z',
+      },
+      // persisted_autoload: {
```
Comment (Monkeychip) [inline ui/mirage/config.js:44]: curious why these are commented out?

**Review by Monkeychip — APPROVED** [review]
Great work, a couple of non-blocking comments.

File: ui/app/templates/components/license-info.hbs (hunk 2 above)
Comment (hashishaw) [reply ui/app/templates/components/license-info.hbs:9]: it was just easier to parse the sections while I was working on removing code

File: ui/mirage/config.js (hunk 3 above)
Comment (hashishaw) [reply ui/mirage/config.js:44]: I was testing with various configurations but wanted to make sure we didn't lose the other options

**Pull request merged**